SingleNumber.js

```diff
@@ -0,0 +1,15 @@
+
+   function findSingle(ar, size)
+    {
+
+        let res = ar[0];
+    for (let i = 1; i < size; i++)
+    res = res ^ ar[i];
+
+    return res;
```

Comment:
Please properly format the code. Although this is not mandatory in JS, but it is a good practice to format the code. It makes the code more readable, and looks more friendly :)